src/RealAntennasProject/Antenna/Encoder.cs

```diff
@@ -30,6 +31,10 @@ public Encoder(string name, int techLevel, float rate, float minEbN0)
 
         public Encoder BestMatching(in Encoder other) => TechLevel > other.TechLevel ? other : this;
         public static Encoder BestMatching(in Encoder a, in Encoder b) => a.BestMatching(b);
+        public static Encoder Get(string name, int level)
+        {
+            return string.IsNullOrEmpty(name) || !All.TryGetValue(name, out Encoder encoder) ? Encoder.GetFromTechLevel(level) : encoder;
+        }
```

Comment:
TBH I would write this as `GetByName` and then in calling code use `GetByName(overrride) ?? GetFromTechLevel(level)`.

Reply:
Alright rewrote it so the discrimination is done when the function is called.